src/Components/TasksPage/TasksTable.tsx

```diff
@@ -0,0 +1,337 @@
+      </Tr>
+    </Tbody>
+  ));
+
+  const getSortParams = (columnIndex: number): ThProps['sort'] => ({
+    sortBy: {
+      index: activeSortIndex,
+      direction: activeSortDirection,
+      defaultDirection: 'asc',
+    },
+    onSort: (_event, index, direction) => {
+      setActiveSortIndex(index);
+      setActiveSortDirection(direction);
+      fetchData(page, perPage, index, direction);
+    },
+    columnIndex,
+  });
+
+  const fetchTaskDetails = (id) => {
+    tasksFetchExecutedTaskDetail(id).then((fetchedTask) => {
+      const newExecutedTasks = [...executedTasks];
+      const index = newExecutedTasks.findIndex((task) => task.id === id);
+      newExecutedTasks[index].details = {
+        initiated_by: fetchedTask.initiated_by,
+        start_time: fetchedTask.start_time,
+        end_time: fetchedTask.end_time,
+        alerts_count: fetchedTask.jobs.reduce((acc, job) => {
+          if (job.status !== 'Success') return acc + 1;
+          return acc + job.results.alert;
+        }, 0),
+      };
+      setExecutedTasks(newExecutedTasks);
+    });
+  };
+
+  const getTaskDetails = (id) => {
+    const task = executedTasks.find((task) => task.id === id);
+    if (task?.details === undefined) {
+      fetchTaskDetails(id);
+      return <Skeleton />;
+    }
+
+    const { details } = task;
+
+    return (
+      <Grid hasGutter>
+        <GridItem span={3}>
+          <TextContent>
+            <Text component="h4" key={`task-details-initiated-${id}`}>
+              Initiated by:
+            </Text>
+            <Text component="p" key={`task-details-initiated-val-${id}`}>
+              {details.initiated_by}
+            </Text>
+          </TextContent>
+        </GridItem>
+        <GridItem span={3}>
+          <TextContent>
+            <Text component="h4" key={`task-details-start-${id}`}>
+              Run start:
+            </Text>
+            <Text component="p" key={`task-details-start-val-${id}`}>
+              {getFormattedTime(details.start_time)}
+            </Text>
+          </TextContent>
+        </GridItem>
+        <GridItem span={3}>
+          <TextContent>
+            <Text component="h4" key={`task-details-end-${id}`}>
+              Run end:
+            </Text>
+            <Text component="p" key={`task-details-end-val-${id}`}>
+              {getEndTime(details.end_time, task.status)} {calculateDurationTime(details.start_time, details.end_time, task.status)}
+            </Text>
+          </TextContent>
+        </GridItem>
+        <GridItem span={3}>
+          <TextContent>
+            <Text component="h4" key={`task-details-systems-${id}`}>
+              Systems with alerts:
+            </Text>
+            <Text component="p" key={`task-details-systems-val-${id}`}>
+              {details.alerts_count}
+            </Text>
+          </TextContent>
+        </GridItem>
+        <GridItem span={12}>
+          <Button
+            variant="secondary"
+            component="a"
```

Comment:
```suggestion
            component="a"
            icon={<ExternalLinkAltIcon />}
```

![image](https://github.com/RedHatInsights/in-place-upgrades-frontend/assets/19702477/116f4711-a585-4a2d-9614-7ef5d19e2314)